packages/fiori/src/ViewSettingsDialog.js

```diff
@@ -279,6 +283,18 @@ class ViewSettingsDialog extends UI5Element {
 		return "";
 	}
 
+	get shouldBuildSort() {
+		return this.sortItems.length;
+	}
+
+	get shouldBuildFilter() {
+		return this.filterItems.length;
+	}
+
+	get hasTwoPages() {
```

Comment:
I think that <code>hasPages</code> is better name for this getter as there could be more than two pages in the future, which wouldn't change the purpose of the function.